vllm/config.py

```diff
@@ -250,6 +250,12 @@ def get_hidden_size(self) -> int:
         return self.hf_text_config.hidden_size
 
     def get_head_size(self) -> int:
+        # TODO remove hard code
+        if hasattr(self.hf_text_config, "model_type"
+                   ) and self.hf_text_config.model_type == 'deepseek_v2':
+            # FlashAttention supports only head_size 32, 64, 128, 256,
```

Comment:
Is this true? According to https://github.com/vllm-project/vllm/blob/99eff67ba9155b5fec9a9abd939e3a29a1b42dce/vllm/attention/backends/flash_attn.py#L16, flash attention supports head size of 192 too. So I think you can remove this, right? And also the related padding code in `deepseek_v2.py` -- that should make it quite a bit simpler :)

Reply:
thanks, i will test it later with the latest flash attn